test/sharness/t0180-pubsub-gossipsub.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+test_description="Test pubsub with gossipsub"
+
+. lib/test-lib.sh
+
+# start iptb + wait for peering
+NUM_NODES=5
+test_expect_success 'init iptb' '
+  iptb init -n $NUM_NODES --bootstrap=none --port=0
+'
+
+test_expect_success "enable gossipsub" '
+  for x in $(seq 0 4); do
+    ipfsi 0 config Pubsub.Router gossipsub
```

Comment:
Should be `ipfsi $x ...`. FYI, you can also use `iptb for-each ipfs config ...`.

Reply:
It's a bit odd that the tests are still passing.

Reply:
oops, good catch!
It's not odd that the test still pass, gossipsub is backwards compatible with floodsub.

Reply:
Ah. Good to know.